doc/md_src_files/090_futureWork.md

```diff
@@ -2,16 +2,56 @@
 
 # Future Work
 
-We intend SSVC to offer a workable baseline from which to improve and refine a vulnerability-prioritization methodology. While the method herein should be functional, we do not claim it is ready for use as is. Therefore, we lay out some aspects of future work that would help make it ready to use. We focus on further requirements gathering, further testing of the reliability of the decision process, and expanding to additional types of stakeholders beyond deployers and suppliers.
+We intend SSVC to offer a workable baseline from which to improve and refine a vulnerability-prioritization methodology.
+We are working to improve SSVC.
+Several of the future work items in this section have [issues associated with them on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues), which is a good place to go to check on progress or help.  
```

Comment:
Current: Several of the future work items in this section have [issues associated with them on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues), which is a good place to go to check on progress or help. 

My comment: Several of the future work items are associated with open issues listed on the SSVC GitHub page (https://github.com/CERTCC/SSVC/issues), which is a good place to go to check on progress and contribute to the project.

Reply:
And only link "open issues"